src/containers/Identity/AddIdentity/View.js

```diff
@@ -12,13 +12,26 @@ const AddIdentity = ({ navigation, actions }) => {
         setName(name);
     };
 
-    const selectIdentity = (id) => {
-    //set flag selected Identity
-   // navigation.navigate('Identity');
+    const handleAdd = () => {
```

Comment:
You can write this a bit nicer.
```
if (!name) return
setIdentity(identities.concat({
  id: uuid(),
  name: name.includes('@') ? name : `${name}@`,
}))
```
The other thing is, you should think about your naming.  Why is the set function named setIdentity and not setIdentities for example?!